lib/gooddata/lcm/actions/collect_ca_metrics.rb

```diff
@@ -13,7 +13,13 @@ class CollectComputedAttributeMetrics < BaseAction
 
       PARAMS = define_params(self) do
         description 'Client Used for Connecting to GD'
-        param :gdc_gd_client, instance_of(Type::GdClientType), required: true
+        param :gdc_gd_client, instance_of(Type::GdClientType), required: false
+
+        description 'Development Client Used for Connecting to GD'
```

Comment:
this is not a development client, but more like a "client used to connecting to development domain"